pygsfit.py

```diff
@@ -1846,6 +1855,25 @@ def init_params(self):
             self.fit_function = gstools.GSCostFunctions.Ff_Gyroresonance_MinimizerOneSrc
             self.update_fit_param_widgets()
 
+        if self.ele_dist == 'thermal':
```

Comment:
I already have a hook to "thermal f-f + gyrores" (lines 1839-1858). Let us merge the two blocks. When setting parmlocal[6]=2 in fast GS code, it uses thermal f-f + gyroresonance. I wish to distinguish this from pure free-free (a cost function needs to be written, but pretty straightforward).